components/plan-build/bin/composite_build_functions.bash

```diff
@@ -141,28 +142,29 @@ _validate_bind_mappings() {
 
     if [[ -f "${resolved}/BINDS" ]]; then
       while read -r line; do
-        IFS== read bind_name exports <<< "${line}"
-        all_binds_for_pkg[$bind_name]="${exports[@]}"
+        IFS="=" read -r bind_name exports <<< "${line}"
+        all_binds_for_pkg[$bind_name]="${exports[*]}"
```

Comment:
In this case `"${exports[*]}"` is a bit misleading, because `exports` will be a scalar, not an array. [`read`](https://www.gnu.org/software/bash/manual/html_node/Bash-Builtins.html#index-read) only creates array variables with the `-a` flag, so this should probably be
```bash
        all_binds_for_pkg[$bind_name]="${exports}"
```
assuming the expectation that every line is of the form
```
bind_name=export
```
And in that case, it should make sense to change `exports` to `export`.

Reply:
Our exports are a little weird... it should be `bind_name="foo bar baz"`, so it really is a scalar.